org.mixedrealitytoolkit.spatialmanipulation/BoundsControl/BoundsControl.cs

```diff
@@ -502,6 +529,29 @@ public SelectExitEvent ManipulationEnded
         // Has the bounds control moved past the toggle threshold throughout the time it was selected?
         private bool hasPassedToggleThreshold = false;
 
+        protected struct LogicImplementation
```

Comment:
For all protected and public types and member...please add a `<summary>` tag describing the item...for our docs